draft-ietf-ppm-dap.md

```diff
@@ -968,7 +956,8 @@ This message consists of:
 * Information used by the Aggregators to determine how to aggregate each report:
 
     * For fixed-size tasks, the Leader specifies a "batch ID" that determines
-      the batch to which each report for this aggregation job belongs.
+      the batch to which each report for this aggregation job belongs. The
```

Comment:
I'm not totally sure "The Leader MAY allocate batch IDs in any order." is necessary -- we could remove this sentence and leave this implicit. Thoughts?

Reply:
I agree it's unnecessary.

Reply:
Removed.